ballerina/tests/resources/documents/subscriptions_with_introspections.graphql

```diff
@@ -0,0 +1,3 @@
+subscription sub { 
+    __typename 
+}
```

Comment:
Shall we add a test case for the following scenario as well?
`{ messages { __typename { name } } }`

Reply:
Sure, I'll add that as well